InventoryComponents/InventoryRelicComponent.gd

```diff
@@ -0,0 +1,15 @@
+extends Node
+class_name InventoryRelicComponent
+
+var held_relics : Array[Relic]
+
+signal held_relics_update(new_relics : Array[Relic])
```

Comment:
i'm not sure it's useful to emit the entire array, maybe just emitting the new relic ? that will be more lightweight for now